contacktbook/BackupClass.cs

```diff
@@ -15,21 +16,30 @@ public class BackupClass
         public void createBackup()
         {
            DataBase db = new DataBase();
-           SQLiteDataReader dataReader = db.selectContact();
-           ListOfConacts contactsList = new ListOfConacts();
-           contactsList.contactList = new List<Contact>();
-    
-       while (dataReader.Read())
+           try
            {
-               Contact cont = new Contact();
-               contactsList.contactList.Add(new Contact("" + dataReader["first_name"], "" + dataReader["second_name"], "" + dataReader["tel1"], "" + dataReader["tel2"], "" + dataReader["email"]));
+               SQLiteDataReader dataReader = db.selectContact();
+               ListOfConacts contactsList = new ListOfConacts();
+               contactsList.contactList = new List<Contact>();
 
-           }
+               while (dataReader.Read())
+               {
+                   Contact cont = new Contact();
+                   contactsList.contactList.Add(new Contact("" + dataReader["first_name"], "" + dataReader["second_name"], "" + dataReader["tel1"], "" + dataReader["tel2"], "" + dataReader["email"]));
+

```

Comment:
Why do you need to add "" in the constructor?
I would refactor these two lines as:
Contact cont = new Contact(dataReader["first_name"], dataReader["second_name"], 
                                             dataReader["tel1"], dataReader["tel2"], dataReader["email"]);
        contactList.contactList.Add(Cont);
)